src/Microsoft.TemplateEngine.Edge/Template/TemplateCreator.cs

```diff
@@ -319,8 +319,8 @@ public void ResolveUserParameters(ITemplate template, IParameterSet templatePara
                 return null;
             }
             IFile config = mountPoint!.FileInfo(info.ConfigPlace);
-            IFile? localeConfig = string.IsNullOrEmpty(info.LocaleConfigPlace) ? null : mountPoint.FileInfo(info.LocaleConfigPlace);
-            IFile? hostTemplateConfigFile = string.IsNullOrEmpty(info.HostConfigPlace) ? null : mountPoint.FileInfo(info.HostConfigPlace);
+            IFile? localeConfig = string.IsNullOrEmpty(info.LocaleConfigPlace) ? null : mountPoint.FileInfo(info.LocaleConfigPlace!);
+            IFile? hostTemplateConfigFile = string.IsNullOrEmpty(info.HostConfigPlace) ? null : mountPoint.FileInfo(info.HostConfigPlace!);
```

Comment:
Isn't `string.IsNullOrEmpty(info.LocaleConfigPlace) ? null :` ensuring that? Reason we even need `!` is because this library is compiled with .NET Standard 2.0 which doesn't have `string.IsNullOrEmpty` properly annotated.